packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/templates/generatedAnswer.css

```diff
@@ -30,6 +28,48 @@
   vertical-align: baseline;
 }
 
+.generated-answer__answer--collapsed {
+  max-height: var(--maxHeight, 250px);
+}
+
+.generated-answer__answer--expanded {
+  max-height: var(100%);
+  margin-bottom: 1rem;
+  overflow: visible;
+}
+
+.generated-answer__answer::after {
+  transition: 0.5s;
+  position: absolute;
+  visibility: hidden;
+  content: '';
+  width: 100%;
+  height: 50px;
+  bottom: 0;
+  left: 0;
+  background-image: linear-gradient(
+    to bottom,
+    transparent,
+    rgba(255, 255, 255, 1)
+  );
+}
+
+.generated-answer__answer--collapsed::after {
+  clip-path: polygon(0 0, 100% 0, 100% 100%, 0% 100%);
+  opacity: 1;
+  visibility: visible;
+}
+
+.generated-answer__answer--expanded::after {
+  clip-path: polygon(0 100%, 100% 100%, 100% 100%, 0% 100%);
+  opacity: 0.2;
```

Comment:
🤷 I was trying to start from the exact same thing as the Smart snippet one, so I copied it from what you had done with the Smart Snippet because it looked nice! lol